components/automate-ui/src/app/entities/automate-settings/automate-settings.effects.ts

```diff
@@ -32,18 +32,8 @@ export class AutomateSettingsEffects {
   @Effect()
   configureSettings$ = this.actions$.pipe(
     ofType(AutomateSettingsActionTypes.CONFIGURE_SETTINGS),
-    mergeMap((action: ConfigureSettings) => {
-      const jobsRequests = [];
-      action.payload.jobs.forEach((job) => {
-        jobsRequests.push(this.requests.configureIngestJob(job));
-      });
-      return forkJoin(jobsRequests).pipe(
-        map((_resp) => new ConfigureSettingsSuccess({})),
-        catchError((error) => of(new ConfigureSettingsFailure(error))));
-    }));
-
-  @Effect()
-  configureSettingsSuccess$ = this.actions$.pipe(
-    ofType(AutomateSettingsActionTypes.CONFIGURE_SETTINGS_SUCCESS),
-    map((_action) => new GetSettings({})));
+    mergeMap((action: ConfigureSettings) => this.requests.configureIngestJobs(action.payload.jobs)),
+    map((_resp) => (new ConfigureSettingsSuccess({}))),
+    catchError((error) => of(new ConfigureSettingsFailure(error))
+    ));
 }
```

Comment:
Removing this effect saved us an unnecessary API call after the new configurations are made, as we've already got them in the form.

Reply:
Nice catch!